### PR TITLE
Upgrade to 2.7.5

### PR DESCRIPTION
**Pull request by dkarpenko**
- Upgrade of Tablesorter to `2.7.5`
- `min.js` files removed from gem
- Patch version incremented only. Is `patch` update of Tablesorter `minor` update for jquery-tablesorter-rails ?

### CODE REVIEW

File: vendor/assets/stylesheets/jquery-tablesorter/theme.blue.css [hunk 1]
```diff
@@ -40,7 +40,7 @@
 	/* white (unsorted) double arrow */
 	/* background-image: url(data:image/gif;base64,R0lGODlhFQAJAIAAAP///////yH5BAEAAAEALAAAAAAVAAkAAAIXjI+AywnaYnhUMoqt3gZXPmVg94yJVQAAOw==); */
 	/* image */
-	/* background-image: url(/assets/jquery-tablesorter/black-unsorted.gif); */
+	/* background-image: url(images/black-unsorted.gif); */
```
Comment (linjunpop) [inline vendor/assets/stylesheets/jquery-tablesorter/theme.blue.css]: Could you run `$ rake jquery_tablesorter:sanitize_image_paths` to update image path?

https://github.com/linjunpop/jquery-tablesorter-rails#update-js-and-css-files

Comment (dkarpenko) [inline vendor/assets/stylesheets/jquery-tablesorter/theme.blue.css]: Shame on me :( 
Updated both submodule and CSSs

**Comment by linjunpop** [comment]
@dkarpenko Thanks. :beers:

**Comment by dkarpenko** [comment]
You are welcome!

**Pull request merged**